packages/wallets/src/components/Timer/Timer.tsx

```diff
@@ -25,9 +26,9 @@ const Timer = ({ countStart = 60, intervalMs = 1000, onComplete }: TTimerProps)
     }, [count, onComplete, resetCountdown, startCountdown]);
 
     return (
-        <WalletText as='p' color='less-prominent' size='sm'>
-            {`${count}s`}
-        </WalletText>
+        <Text as='p' color='less-prominent' size='sm'>
+            <Localize i18n_default_text='{{counts}}s' values={{ count }} />
```

Comment:
Hi @george-usynin-deriv,  don't be a victim like I was 😁 
```suggestion
            <Localize i18n_default_text='{{count}}s' values={{ count }} />
```

Reply:
ahah @jim-deriv already have this raised bug :)
thanks for checking!